data/scripts.js

```diff
@@ -636,14 +636,7 @@ exports.BattleScripts = {
 		return isValid;
 	},
 	sampleNoReplace: function (list) {
-		let length = list.length;
-		let index = this.random(length);
-		let element = list[index];
-		for (let nextIndex = index + 1; nextIndex < length; index += 1, nextIndex += 1) {
```

Comment:
(`++index` is 5% faster than `index += ` in Node 4.0.0; `index++` is now only 1% slower)